catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/IcebergTable.java

```diff
@@ -222,4 +233,39 @@ protected IcebergTable internalBuild() {
   public static Builder builder() {
     return new Builder();
   }
+
+  private static Map<String, String> buildReservedProperties(TableMetadata table) {
```

Comment:
could you move this logic to a helper or util class with apache header and add to LICENCE?

Reply:
@FANNG1 only move this method?

Reply:
fixed.